### PR TITLE
patch: Speed up Core's docker build

### PR DESCRIPTION
**Pull request by vipulgupta2048**
```
➜  bro git:(master) ✗ sudo docker build . -t balena-cli:latest
Sending build context to Docker daemon  2.048kB
Step 1/5 : FROM node:12-bullseye
 ---> 99ea8906ba9c
Step 2/5 : WORKDIR /usr/src/app
 ---> Using cache
 ---> 481b8f063fa6
Step 3/5 : RUN curl -L https://github.com/balena-io/balena-cli/releases/download/v12.51.1/balena-cli-v12.51.1-linux-x64-standalone.zip -o balena-cli.zip     && unzip balena-cli.zip     && rm balena-cli.zip
 ---> Using cache
 ---> 4e36cd5d926e
Step 4/5 : ENV PATH="/usr/src/app/balena-cli/:${PATH}"
 ---> Using cache
 ---> 14dc16a84c2e
Step 5/5 : CMD ["balena", "--version"]
 ---> Using cache
 ---> 93e83fc800a5
Successfully built 93e83fc800a5
Successfully tagged balena-cli:latest


➜  bro git:(master) ✗ sudo docker run -it balena-cli:latest
12.51.1
```

### CODE REVIEW

File: core/Dockerfile.template [hunk 1]
```diff
@@ -24,7 +24,11 @@ COPY --from=npm-install /tmp/node ./
 
 RUN install_packages python3 make build-essential openssh-client
 
-RUN npm install balena-cli@12.51.1 -g --production --unsafe-perm
+RUN curl -L https://github.com/balena-io/balena-cli/releases/download/v12.51.1/balena-cli-v12.51.1-linux-x64-standalone.zip -o balena-cli.zip \
```
Comment (klutchell) [inline core/Dockerfile.template:27]: @vipulgupta2048 this binary is for x64 and won't run on the Fin.

**Review by klutchell — CHANGES_REQUESTED** [review]
This binary is for x64 and won't run on the Fin.

File: core/Dockerfile.template (hunk 1 above)
Comment (vipulgupta2048) [reply core/Dockerfile.template:27]: Ahh so that's why we have an NPM install in place here, and so do you in your balena-cli base image. I gotcha.

Comment (vipulgupta2048) [reply core/Dockerfile.template:27]: Will wait for the balena-cli to releae the apt installation method.

**Pull request closed without merging**